### DISCUSSION TITLE
How to create a message with content_type 'cards' on dashboard using API

### DISCUSSION THREAD

**Discussion by AdsonCicilioti** [chatwoot/chatwoot#7890] [Q&A]
I'm working on an application nodejs and I would like to know what JSON data structure I need to provide to produce a card-type message using the rich metadata that I previously obtain from a url?

**Comment by MichelN89** [comment]
I think this is what you need: https://www.chatwoot.com/docs/product/others/interactive-messages

**Reply by AdsonCicilioti** [reply]
Yes, I found this part of the documentation and I'm sorry, I forgot to say that I want to create this type of message on the dashboard.  I have an api that captures messages from leads that come from ads, and I would like the first message coming from the ad to come in card format, bringing references from the ad so that the multi-service team can identify which ad this lead is coming from.
